submissions/HannaSyn/Popup/index.html

```diff
@@ -0,0 +1,87 @@
+                <p>Apps</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/blogs.png" alt="blogs">
+                <p>Blogs</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/chat.png" alt="chat">
+                <p>Chat</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/calendar.png" alt="calendar">
+                <p>Calendar</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/files.png" alt="files">
+                <p>Files</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/photos.png" alt="photos">
+                <p>Photos</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/videos.png" alt="videos">
+                <p>Videos</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/documents.png" alt="documents">
+                <p>Documents</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/contacts.png" alt="contacts">
+                <p>Contacts</p>
+              </a>
+                <input class="menu-popup__input" type="checkbox" id="more-btn" />
+                <label class="menu-popup__label" for="more-btn"></label>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/maps.png" alt="maps">
+                <p>Maps</p>
+              </a>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/search.png" alt="search">
+                <p>Search</p>
+              </a>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/classroom.png" alt="classroom">
+                <p>Classroom</p>
+              </a>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/mars.png" alt="mars">
+                <p>Mars</p>
+              </a>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/promo.png" alt="promo">
+                <p>Promo</p>
+              </a>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/tanslation.png" alt="translation">
+                <p>Translation</p>
+              </a>
+            </div>
+          </div>
+        </li>
+        <li class="navigation__link">
+          <a  href="#"><img src="assets/bell.svg"
+              alt="bell"></a>
+        <li  class="navigation__link link">
+          <a  href="https://github.com/HannaSyn/kottans-frontend" class="link__user"><img src="assets/user.svg"
+              alt="user"></a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main">
+    <figure class="bg">
```

Comment:
You can add `background-image` to `<main>` and remove these unnecessary tags.)

Reply:
I have read many articles about this subject and decided that it's better solution - to use tags. It's  better to find tag in html, then background in css.

Reply:
If this image is just decorative it must be removed from HTML markup. Every HTML element can affect the page: in the case of screenreaders you can hide this image, but in the case of search(google) bot you risk to be removed from the search results because the search bot parses your HTML and every image has a huge influence on parse result if something doesn't like to him with this image he can downgrade your site in search results.

Reply:
Fixed.